tests/test_clic2020.py

```diff
@@ -0,0 +1,33 @@
+import pathlib
+import shutil
+
+import pytest
+
+from neuralcompression.data import CLIC2020
+
+
+@pytest.fixture(autouse=True)
+def remove_data_directory():
+    yield
+
+    if pathlib.Path("data").exists():
+        shutil.rmtree("data")
+
+
+@pytest.fixture
+def test_data():
+    return CLIC2020("data", "test")
+
+
+class TestCLIC2020:
+    def test___getitem__(self, test_data):
+        with pytest.raises(IndexError):
+            assert test_data[0]
+
+    def test___len__(self, test_data):
+        assert len(test_data) == 0
+
+    def test_download(self, test_data):
+        test_data.download()
+
+        assert len(test_data) == 60
```

Comment:
We shouldn't include downloads in CI - we can just trust that you ran it and it worked. See the reference linked above for an example of creating test data and monkeypatching.